src/model/image_info.js

```diff
@@ -298,8 +305,13 @@ export default class ImageInfo {
         this.can_annotate = response.perms.canAnnotate;
         if (typeof response.meta.imageAuthor === 'string')
             this.author = response.meta.imageAuthor;
-        if (typeof response.meta.imageName === 'string')
+        if (typeof response.meta.imageName === 'string') {
```

Comment:
Generating a short name here makes sense
but we should in that case remove the code in ``header.js``
This is now duplicated